festim/mesh/mesh_from_xdmf.py

```diff
@@ -0,0 +1,62 @@
+from dolfinx.io import XDMFFile
+from mpi4py import MPI
+import festim as F
+
+
+class MeshFromXDMF(F.Mesh):
+    """
+    Mesh read from the XDMF files
+
+    Args:
+        volume_file (str): path to the volume file
+        facet_file (str): path to the facet file
+        mesh_name (str, optional): name of the mesh in the XDMF file. Defaults to "Grid".
+        meshtags_name (str, optional): name of the meshtags in the XDMF file. Defaults to "Grid".
+
+    Attributes:
+        volume_file (str): path to the volume file
+        facet_file (str): path to the facet file
+        mesh_name (str, optional): name of the mesh in the XDMF file. Defaults to "Grid".
+        meshtags_name (str, optional): name of the meshtags in the XDMF file. Defaults to "Grid".
+        mesh (fenics.mesh.Mesh): the mesh
+    """
+
+    def __init__(
+        self, volume_file, facet_file, mesh_name="Grid", meshtags_name="Grid"
```

Comment:
Just a comment, this doesn't allow to have different meshtags names in the volume file and the facet file